cmd/manager/main.go

```diff
@@ -32,5 +32,5 @@ func main() {
 	if err != nil {
 		log.Fatal("Error building kubeconfig", err)
 	}
-	sharedmain.MainWithConfig(signals.NewContext(), "serving-operator", cfg, knativeserving.NewController)
+	sharedmain.MainWithConfig(signals.NewContext(), "operator", cfg, knativeserving.NewController)
```

Comment:
Why do you change the name here?

Reply:
Because the component name `serving-operator` will be passed to the prometheus exporter as a prefix in the metric name, but the symbol `-` is not allowed in the prometheus metric name. Meanwhile, single-word application/component name is recommended by prometheus and all other components in knative serving have single-word names.

Reply:
Is serving.operator supported? I lean on to keep the full name.

Reply:
Only letters, digits and underscores are supported, and single-word prefix is recommended. See  https://prometheus.io/docs/practices/naming/

Since all knative serving components have single-word names, e.g. webhook, activator, autoscalar, it makes sense to align with the current naming pattern.

Reply:
Good catch @jihuin I noticed the same, and I know that **ALL** other knative components _NOW_  using `_` underscore.  For instance:
* https://github.com/knative/eventing/commit/36058f4fe735902265b63ab43c3f82ff289d07ef

Since we have more than one operator, let's please use `serving_operator` !

Reply:
We can use the underscore.

Reply:
@jihuin can you sent a PR to use `serving_operator` with underscore ?

Reply:
Changed to `serving_operator`